v1/ast/visit.go

```diff
@@ -575,14 +745,6 @@ func (vis *VarVisitor) Clear() *VarVisitor {
 	return vis
 }
 
-// ClearOrNew returns a new VarVisitor if vis is nil, or else a cleared VarVisitor.
-func (vis *VarVisitor) ClearOrNew() *VarVisitor {
```

Comment:
Theoretically, someone external could use this method? Would it hurt to keep it, just in case, with a deprecated comment?

Reply:
Fair enough! I'll add it back 👍